src/script.js

```diff
@@ -1,167 +1,210 @@
 import * as THREE from 'three'
 import { OrbitControls } from 'three/examples/jsm/controls/OrbitControls.js'
 import * as dat from 'lil-gui'
-import FakeGlowMaterial from './FakeGlowMaterial.js'
-
-/**
- * Scene
- */
-const canvas = document.querySelector('canvas.webgl')
-const scene = new THREE.Scene()
-
-/**
- * ScreenResolution
- */
-const screenRes = {
-  width: window.innerWidth,
-  height: window.innerHeight
+import WebGPURenderer from 'three/addons/renderers/webgpu/WebGPURenderer.js';
+import { FakeGlowMaterial as FakeGlowMaterialWebGPU } from './FakeGlowMaterialWebGPU.js'
+import FakeGlowMaterial from './FakeGlowMaterial.js';
+
+const params = {
+  falloff: .1,
+  glowInternalRadius: 6,
+  glowColor: new THREE.Color('#8039ea'),
+  glowSharpness: .5,
+  opacity: 1,
+  toneMappingExposure: 1,
+  depthTest: false,
 }
 
-window.addEventListener('resize', () => {
-  screenRes.width = window.innerWidth
-  screenRes.height = window.innerHeight
+function initScene(isWebGPU) {
+  /**
+   * Scene
+   */
+  const canvas = document.querySelector(isWebGPU ? 'canvas.webgpu' : 'canvas.webgl')
+  const scene = new THREE.Scene()
+
+  /**
+   * ScreenResolution
+   */
+  const screenRes = {
+    width: window.innerWidth,
+    height: window.innerHeight / 2
+  }
+
+  window.addEventListener('resize', () => {
+    screenRes.width = window.innerWidth
+    screenRes.height = window.innerHeight / 2
+
+    camera.aspect = screenRes.width / screenRes.height
+    camera.updateProjectionMatrix()
+
+    renderer.setSize(screenRes.width, screenRes.height)
+    renderer.setPixelRatio(Math.min(window.devicePixelRatio, 1))
+  })
 
-  camera.aspect = screenRes.width / screenRes.height
-  camera.updateProjectionMatrix()
+  /**
+   * Camera
+   */
+  const camera = new THREE.PerspectiveCamera(
+    35,
+    screenRes.width / screenRes.height,
+    1,
+    1000
+  )
+  camera.position.set(0, 0, 6)
+  scene.add(camera)
+
+  /**
+   * Controls
+   */
+  const controls = new OrbitControls(camera, canvas)
+  controls.enableDamping = true
+  controls.maxDistance = 8
+  controls.minDistance = 2
+  controls.maxPolarAngle = Math.PI / 1.7
+  controls.minPolarAngle = 1.1
+  controls.autoRotate = true
+  controls.target.set(0, -0.3, 0)
+
+  /**
+   * Lights
+   */
+  const light = new THREE.DirectionalLight()
+  light.intensity = 1
+  light.position.set((isWebGPU ? -1 : 1) * -20, (isWebGPU ? -1 : 1) * 20, (isWebGPU ? -1 : 1) * 50)
```

Comment:
I don't know why `DirectionLight` on `skybox` in `webgl` and `webgpu` is inversed. I fixed it by mutiply '-1' in `webgpu`. However light is not right on `MeshPhysicalMaterial`